projects/laji/src/app/+theme/theme-routing.module.ts

```diff
@@ -92,6 +93,8 @@ const routes: Routes = [
   {path: 'pinkka',  pathMatch: 'full', component: PinkkaComponent, data: {title: 'navigation.pinkka'}},
   {path: 'publications',  pathMatch: 'full', component: BibliographyComponent, data: {title: 'finbif-bib.title'}},
   {path: 'hyonteisopas',  pathMatch: 'full', component: InsectGuideComponent, data: {title: 'navigation.hyonteisopas'}},
+  {path: 'dataset-metadata', pathMatch: 'full', component: DatasetMetadataComponent, data: {title: 'theme.dataset-metadata'}},
+  {path: 'dataset-metadata/:collectionId', pathMatch: 'full', component: DatasetMetadataComponent, data: {title: 'theme.dataset-metadata'}},
```

Comment:
Could you move the `dataset-metadata/*` to a separate lazy-loaded module? That way the code isn't loaded when user navigates to other theme pages. The theme page is already quite bloated and we should try to keep the components isolated. All the new imports in `theme.module` could be in that module.